dev/com.ibm.ws.springboot.fat20.concurrency.app/src/main/java/com/example/demo/AppRunner.java

```diff
@@ -0,0 +1,66 @@
+package com.example.demo;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.fail;
+
+import java.util.concurrent.CompletableFuture;
+
+import javax.naming.InitialContext;
+import javax.naming.NamingException;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.stereotype.Component;
+
+@Component
+class AppRunner implements CommandLineRunner {
+
+	private final static Logger logger = LoggerFactory.getLogger(AppRunner.class);
+
+	private static ConcurrencyTasks concurrencyApplicationConfig = new ConcurrencyTasks();
```

Comment:
This should not be `static`.  It should be `final` and assigned by the constructor with the injected value.